ambry-cloud/src/main/java/com/github/ambry/cloud/HelixVcrClusterSpectator.java

```diff
@@ -68,17 +66,12 @@ public void spectate() throws Exception {
     // zk connection fails on both data centers, then things like replication between data centers might just stop.
     // For now, since we have only one fabric in cloud, and the spectator is being used for only cloud to store replication, this will work.
     // Once we add more fabrics, we should revisit this.
-    for (DcZkInfo dcZkInfo : dataCenterToZkAddress.values()) {
-      // only handle vcr clusters for now
-      if (dcZkInfo.getReplicaType() == ReplicaType.CLOUD_BACKED) {
-        HelixManager helixManager =
-            helixFactory.getZkHelixManagerAndConnect(cloudConfig.vcrClusterName, selfInstanceName,
-                InstanceType.SPECTATOR, dcZkInfo.getZkConnectStrs().get(0));
+    HelixManager helixManager =
```

Comment:
Nit: can you please remove some unused imports from this file. They became unused because this PR removes some code from this file.